dbt_subprojects/tokens/models/transfers_and_balances/kaia/tokens_kaia_balances_daily_agg_base.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        schema = 'tokens_kaia',
+        alias = 'balances_daily_agg_base',
+        file_format = 'delta',
+        materialized='incremental',
+        incremental_strategy='merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['block_number', 'unique_key'],
```

Comment:
```suggestion
        unique_key = ['day', 'block_number', 'unique_key'],
        partition_by = ['day']
```

Reply:
testing performance with this added. checked CI table and each day has ~1mil rows, so reasonable partition size